<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Console</class>
 <widget class="QMainWindow" name="Console">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>472</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ENDL3SS Console</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="ConsoleScreen">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>451</width>
      <height>351</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="CommandLine">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="GoButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>370</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Go!</string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
